pages/Industrial-Miners.md

```diff
@@ -0,0 +1,15 @@
+The Industrial Miners are part of the [Basic Machines](https://github.com/Slimefun/Slimefun4/wiki/Basic-Machines) category.  
+
+These machines will dig up ores from the area underneath them and put them in a chest.  
+You have to put fuel in the chest before it is able to start working, more info on each fuel can be found in the Slimefun guide.  
+
+![Industrial Miners setup](https://raw.githubusercontent.com/Slimefun/Slimefun4-Wiki/master/images/multiblock-industrial-miners.png)
+
+On the image you can see the setup of the Industrial Miner and the Advanced Industrial Miner respectively.
+
```

Comment:
A GIF of their animation would be super cool but it isn't necessary :o

Reply:
maybe some other time if thats okay